Carey's advice addition

Aurora808 opened the pull request:
Added another page of advice. Look good?

redrambles commented:
Hey @Aurora808 !

This is really good - there's just one change that we need you to make. You have added a `advice/index.html` file - but we don't need this, so please delete it. What we need instead is for you to add a single line to the existing index.html file that is in the root of the project. This line will link to your advice page.

Once you have made those changes - please add and commit and push up your `careys_advice` branch once more, which will automatically update this pull request. 

Then we should be all good to merge in your code! Let us know if you have any questions. :)

Cheers,
Ann :)

Aurora808 commented:
Hi Ann,
thanks for processing. 
I believe I took care of the requests. I deleted the extra file and added the line all within my branch in github. I’m not sure what you meant by “push up” one more time (and perhaps because I did it all directly in Github I didn’t need to push up again?), but I did request another pull. So I think this is what you wanted? If not - please let me know. 
I’m going to review the Github lessons another time today. I’m just following directions, but not clear on all that I’ve done - even if I did it correctly. :)


Carey

On Saturday, November 16, 2019, 1:15 PM, Ann Cascarano <notifications@github.com> wrote:


Hey @Aurora808 !

This is really good - there's just one change that we need you to make. You have added a advice/index.html file - but we don't need this, so please delete it. What we need instead is for you to add a single line to the existing index.html file that is in the root of the project. This line will link to your advice page.

Once you have made those changes - please add and commit and push up your careys_advice branch once more, which will automatically update this pull request.

Then we should be all good to merge in your code! Let us know if you have any questions. :)

Cheers,
Ann :)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.

acodeinprogress commented:
Hey @Aurora808,

Great job and thanks for making those changes! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
